src/util/psbt/serialize.rs

```diff
@@ -205,16 +205,17 @@ impl Serialize for schnorr::SchnorrSig  {
 
 impl Deserialize for schnorr::SchnorrSig {
     fn deserialize(bytes: &[u8]) -> Result<Self, encode::Error> {
-        match schnorr::SchnorrSig::from_slice(&bytes) {
-            Ok(sig) => Ok(sig),
-            Err(schnorr::SchnorrSigError::InvalidSighashType(flag)) => {
-                Err(encode::Error::from(psbt::Error::NonStandardSigHashType(flag as u32)))
+        schnorr::SchnorrSig::from_slice(&bytes).map_err(|e| match e {
```

Comment:
I feel that one of these styles would be easier to understand by readers:

```rust
schnorr::SchnorrSig::from_slice(&bytes)
    .map_err(|e| match e {
       ...
    })
```

```rust
schnorr::SchnorrSig::from_slice(&bytes)
    .map_err(|e|
        match e {
            ...
        }
    )
```

```rust
schnorr::SchnorrSig::from_slice(&bytes)
    .map_err(|e|
        match e {
           ...
        }
    )
```

Reply:
thank you for review.
fixed code!